features/frameworks/greentea-client/source/greentea_test_env.cpp

```diff
@@ -270,7 +268,7 @@ inline void greentea_write_string(const char *str)
  *        This function writes an integer value from the target
  *        to the host. The integer value is converted to a string and
  *        and then written character by character directly to the serial
- *        port using the greentea_serial, Rawserial object.
+ *        port using the greentea_serial, GreenteaSerial object.
```

Comment:
references to serial throughout this file should be cleaned up - they all should just reference "console".

Reply:
Please review [this](https://github.com/ARMmbed/mbed-os/compare/10b2d1c8aa95b39231cb96777f816895ce9538bb..72e390ca702f6528c2c52c576808d7a4decbbfd5).